src/components/projects/view/leave-feedback-modal.svelte

```diff
@@ -0,0 +1,87 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import Dialog from 'ui/dialog.svelte';
+  import TextField from 'ui/text-field.svelte';
+  import CheckChipGroup from 'ui/check-chip-group.svelte';
+  import Modal from 'ui/modal.svelte';
+
+  export let isOpen = false;
+  export let activity;
+  export let application;
+  export let competences;
+
+  let value = {
+    competences: [],
+    answers: activity == null ? [] : new Array(activity.feedback_questions.length).fill(''),
+  };
+
+  const competenceItems = competences.map(({ id }) => ({ value: id, checked: false }));
+  const competenceLabels = competences.map(({ name }) => name.toLowerCase());
+
+  function processCheck({ detail: competence }) {
+    if (competence.checked) {
+      value.competences.push(competence.value);
+      value.competences = value.competences;
+    } else {
+      value.competences = value.competences.filter(comp => comp !== competence.value);
+    }
+  }
+
+  const dispatch = createEventDispatcher();
+</script>
+
+<Modal bind:isOpen>
+  {#if activity != null}
+    <Dialog
+      classname="feedback-modal"
+      title="Leave Feedback on {activity.name}"
+      closeCallback={() => isOpen = false}
+    >
+      <div class="content" slot="content">
+        <p>You are just one step away from receiving innopoints!</p>
+        <p>
+          Be honest – we read this feedback, and it will not
+          affect the amount of innopoints you receive!
+        </p>
+        <form>
+          <div class="form-field">
+            <label for="competences">
+              What competences have you developed (up to 3)?
+              <span class="required">*</span>
+            </label>
+            <CheckChipGroup
+              classname="competence-chips"
+              items={competenceItems}
+              name="competences"
+              labels={competenceLabels}
+              maxChecked={3}
```

Comment:
When leaving feedback, Andrey mentioned that students can select more than 3

Reply:
That's bullshit, because he always changes his mind. Let's keep track of a list of explicit questions we have for him, and ask them all